app.json

```diff
@@ -32,6 +32,9 @@
     "ENVIRONMENT": {
       "required": true
     },
+    "ERROR_PAGE_URL": {
+      "required": true
```

Comment:
Setting these as required, but not setting a value or a generator, means they inherit from the parent app in review apps. Which is maybe ok, but it does mean that if we create a branch with some edits to the error pages, then run the management command, it'll update the pages for the parent app as well as the review app. Which is probably no big deal, but might be a bit unexpected, as in most other respects (eg the database) the review apps fork their state from the parent rather than share it.

Creating a new S3 bucket for each review app, uploading the error pages, pointing the review app at those pages, and then destroying them all when the review app is closed, is probably a bit of a pain to set up though?

Reply:
I created a separate bucket for dev error pages to be generated into. The review apps will inherit that. It's not perfect, but it feels fine. We have larger issues with reproducible/separate development environments, and this will need to be addressed as part of that larger piece of work, I think.

Reply:
👍